src/v2/controllers/FormController.js

```diff
@@ -71,17 +88,18 @@ export default Controller.extend({
 
   handleFormSave (model) {
     const formName = model.get('formName');
-    const actionFn = this.options.appState.get('currentState')[formName];
-    if (!_.isFunction(actionFn)) {
+    const idx = this.options.appState.get('idx');
+    if (!idx['neededToProceed'][formName]) {
```

Comment:
Why don't we need both `idx['neededToProceed'][formName]` and `_.isFunction(actionFn)`?

Reply:
Previously `actionFn ` pointed to `this.options.appState.get('currentState')[formName]` and now it points to `idx['neededToProceed'][formName]`. I have just removed the need to hold `idx['neededToProceed'][formName]` in a const and used it directly in the if block as per [this](https://github.com/okta/okta-signin-widget/pull/1033#discussion_r383384294) request.